Dockerfile

```diff
@@ -1,14 +1,13 @@
 FROM alpine:3.13
 
-# Install packges needed
+# Install required packages
 RUN apk --no-cache add ca-certificates curl bash jq py3-pip && \
     pip install awscli
 
-COPY ecs-deploy /ecs-deploy
-RUN chmod a+x /ecs-deploy
+COPY ecs-deploy /usr/local/bin/ecs-deploy
```

Comment:
This is better, yes. But there's no chmod. :( How do you know it is executable?

Reply:
I don't know if you can find out on Github. But you could clone the repo and see the execute bit on the file.

Reply:
Actually, there is a way on Github.

![image](https://user-images.githubusercontent.com/3172830/235464175-1181dffb-275d-4d94-bf10-a81d5ecd0ed9.png)

Reply:
I would prefer having the `chmod` command in there.